### PR TITLE
Copyedit: “Double-clicking” -> “Double-clicking”

### PR DESCRIPTION
**Pull request by hughlilly**
The phrase “double-clicking” should almost always be hyphenated.

### CODE REVIEW

**Review by Tyriar — DISMISSED** [review]

**Review by andreamah — CHANGES_REQUESTED** [review]

File: src/vs/workbench/contrib/search/browser/search.contribution.ts [hunk 1]
```diff
@@ -300,11 +300,11 @@ configurationRegistry.registerConfiguration({
 			enum: ['selectWord', 'goToLocation', 'openLocationToSide'],
 			default: 'goToLocation',
 			enumDescriptions: [
-				nls.localize('search.searchEditor.doubleClickBehaviour.selectWord', "Double clicking selects the word under the cursor."),
-				nls.localize('search.searchEditor.doubleClickBehaviour.goToLocation', "Double clicking opens the result in the active editor group."),
-				nls.localize('search.searchEditor.doubleClickBehaviour.openLocationToSide', "Double clicking opens the result in the editor group to the side, creating one if it does not yet exist."),
+				nls.localize('search.searchEditor.doubleClickBehaviour.selectWord', "double-clicking selects the word under the cursor."),
```
[the remark this answers is not in the document]
Comment (hughlilly) [reply src/vs/workbench/contrib/search/browser/search.contribution.ts]: Oops, sorry! Fixed now.

**Review by andreamah — DISMISSED** [review]
thanks! 😊

**Comment by sbatten** [comment]
I still see places in the code where the hyphenated version is not used. Can we tackle them all?

**Comment by Tyriar** [comment]
@sbatten if it's in code/comments it doesn't matter, only user facing matters

**Review by Tyriar — APPROVED** [review]

**Review by TylerLeonhardt — APPROVED** [review]

**Review by andreamah — APPROVED** [review]

**Pull request merged**